dotnet/targets/Xamarin.Shared.Sdk.MultiTarget.template.targets

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <ItemGroup Condition=" '$(TargetPlatformIdentifier)' == '$(_PlatformName)' and $([MSBuild]::VersionEquals($(TargetFrameworkVersion), '6.0')) ">
+    <KnownFrameworkReference
+        Update="Microsoft.$(_PlatformName)"
+        DefaultRuntimeFrameworkVersion="@NET6_NUGET_VERSION_NO_METADATA@"
+        LatestRuntimeFrameworkVersion="@NET6_NUGET_VERSION_NO_METADATA@"
+        TargetingPackVersion="@NET6_NUGET_VERSION_NO_METADATA@"
+  />
```

Comment:
This seems to match what we do in Android here:

https://github.com/xamarin/xamarin-android/blob/7c9c24b3614710614c5512d7a3b8272065270dc2/src/Xamarin.Android.Build.Tasks/Microsoft.NET.Sdk.Android/WorkloadManifest.in.targets#L9-L15

A question for later: can we use `**FromWorkload**` here? That would prevent us from having to hardcode version numbers in this file.

Reply:
be careful using **FromWorkload** along with any aliasing

Reply:
`**FromWorkload**` was what got us into this mess in the first place...